### PR TITLE
Addresses Refresh imports 07-2023

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Imports refreshed in preparation of 07-2023 CL release.

Cursory check reveals expected updates.

### CODE REVIEW

**Review by aleixpuigb — APPROVED** [review]

File: src/ontology/imports/merged_import.owl [hunk 1]
```diff
@@ -116500,6 +116537,7 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/PR_000001006> "PR:000001006")
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/PR_000001006> "Category=gene. Requested by=CL.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/PR_000001006> "receptor-type tyrosine-protein phosphatase C")
+AnnotationAssertion(<ttps://w3id.org/semapv/vocab/crossSpeciesExactMatch> <http://purl.obolibrary.org/obo/PR_000001006> <http://purl.obolibrary.org/obo/PR_P08575>)
```
Comment (gouttegd) [inline src/ontology/imports/merged_import.owl:116540]: Something wrong happened here: the IRI for the annotation property should be `https://w3id.org/semapv/vocab/crossSpeciesExactMatch`, not `ttps://w3id.org/semapv/vocab/crossSpeciesExactMatch`.

This is probably an error in the upstream import, so I won’t block the PR for that, but this error must be noted. I’ll try tracking the provenance of the error.

**Review by gouttegd — APPROVED** [review]

File: src/ontology/imports/merged_import.owl (hunk 1 above)
Comment (bvarner-ebi) [reply src/ontology/imports/merged_import.owl:116540]: Thanks for investigating, @gouttegd.

Comment (gouttegd) [reply src/ontology/imports/merged_import.owl:116540]: FYI: the error came from the PR slim (https://github.com/obophenotype/pro_obo_slim/issues/32). It’s fixed now, so the next time the imports are refreshed those incorrect IRIs will go away automatically.

Comment (bvarner-ebi) [reply src/ontology/imports/merged_import.owl:116540]: Thank you, @gouttegd. Unless advised to do sooner, the next import refresh will happen after the July release and before the August release.

Comment (anitacaron) [reply src/ontology/imports/merged_import.owl:116540]: If the issue is already fixed, I suggest refreshing the imports again today not to propagate the issue.

Comment (gouttegd) [reply src/ontology/imports/merged_import.owl:116540]: I agree. I don’t like the idea of making a release with known errors in it.

**Pull request merged**